### PR TITLE
IDE-2155 Add function to dynamically get liferay extension point

### PR DESCRIPTION
**Pull request by lovettli**

### CODE REVIEW

**Comment by gamerson** [comment]
good job @lovettli  this looks pretty good, i'll review this fully in a few days.  thanks

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java [hunk 1]
```diff
@@ -149,6 +149,36 @@ private static IPath getBladeCLIPath() throws BladeCLIException
         return projectTemplateNames;
     }
 
+    public static String[] getIntegrationPoints() throws BladeCLIException
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/BladeCLI.java]: Don't add these to BladeCLI.  BladeCLi should just mimic the actually functionalities of the blade cli program.  These two methods should be add in the class that needs the integration points.

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by lovettli** [comment]
Do we need add these two methods into ProjectUtil or other Util Class? Actually i think if we don't need add these two static methods into BladeCLi, we can just remove them and use it via this way "new ServicesCommand().execute();" what do you think?

**Pull request closed without merging**